core/src/main/java/org/moditect/Moditect.java

```diff
@@ -35,100 +34,78 @@ public class Moditect {
 
     public static void main(String[] args) throws Exception {
         CliArgs cliArgs = new CliArgs();
-        new JCommander( cliArgs, args );
+        new JCommander(cliArgs, args);
 
-        new AddModuleInfo( null, null, null, null, cliArgs.outputDirecory, cliArgs.jvmVersion, cliArgs.overwriteExistingFiles, cliArgs.timestamp ).run();
+        new AddModuleInfo(null, null, null, null, cliArgs.outputDirecory, cliArgs.jvmVersion, cliArgs.overwriteExistingFiles, cliArgs.timestamp).run();
     }
 
     @Parameters(separators = "=")
     private static class CliArgs {
 
-        @Parameter(
-            names = "--module-info",
-            required = true,
-            description = "Path to the module-info.java descriptor",
-            converter = PathConverter.class
-        )
+        @Parameter(names = "--module-info", required = true, description = "Path to the module-info.java descriptor", converter = PathConverter.class)
```

Comment:
Is this actually more readable? Not that I'm interested in having a formatting style discussion, but could we configure it so it doesn't join wrapped lines? At least there's such an option in the Eclipse formatter. If it's not doable here for some reason, let's go with the proposed change.

Reply:
I took the formatting configuration from the moditect-org-parent project

Reply:
Ok, can we then perhaps add the "don't join wrapped lines" config there?

Reply:
As far as I can tell it's already there

```
<setting id="org.eclipse.jdt.core.formatter.join_wrapped_lines" value="false"/>
```